syntheseus/tests/reaction_prediction/cli/test_eval.py

```diff
@@ -78,7 +97,43 @@ def get_model_results(remove_duplicates: bool = True, **kwargs):
         assert results_with_repeats == DummyModel.RESULTS
 
 
-def test_print_and_save():
+@pytest.mark.parametrize("forward", [False, True])
+@pytest.mark.parametrize("num_dataset_truncation", [2, 3, 4])
+@pytest.mark.parametrize("num_top_results", [2, 3, 4, 5])
+def test_compute_metrics(
+    forward: bool, num_dataset_truncation: int, num_top_results: int, tmp_path: Path
+) -> None:
```

Comment:
Maybe add a docstring to this function describing what part of the eval script this tests (both explicitly and implicitly)?